docker-compose.yml

```diff
@@ -43,6 +43,13 @@ services:
       - ./server:/apps/warehouse/server
       - ./package.json:/apps/warehouse/package.json
       - ./npm-shrinkwrap.json:/apps/warehouse/npm-shrinkwrap.json
+    depends_on:
+      - db
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://web:3000/api/StoreModels"]
```

Comment:
@harshadyeola - your comment about `exit` something came in my email but disappeared from the PR ... did you change your mind?